addons/OPT/GPS/fn_clientInit.sqf

```diff
@@ -52,4 +52,23 @@
 	// MiniGPS Steuerung Init
 	[] call FUNC(miniGPS);
 
+	// Kontinuierlicher Neustart des GPS System, Löschung alle Marker und Neuerstellung
+	[{
+		// alle Marker Löschen der Einheiten
+		allPlayers apply 
+		{
+			[_x] call FUNC(removeUnitFromGPS);
+		};
+
+		// Marker neu erstellen für alle
+		allPlayers apply 
+		{
+			if (CLib_Player != _x && [_x] call FUNC(isUnitVisible)) then 
+			{
+				[_x] call FUNC(addUnitToGPS);
+			};
+		};	
```

Comment:
Warum iterierst du zweimal hintereinander und separat über `allPlayers`? Pack doch alle Aktionen in eine Schleife zusammen, dann musst du nur einmal das Array durch gehen.

Reply:
weil ich erster alle getrennt Löschen will und dann erst alle neu erstellen will.